pkg/image/manifest.go

```diff
@@ -114,25 +213,21 @@ func NewRegistryList(repoConfig, k8sVersion string) (*RegistryList, error) {
 }
 
 // getImageConfigs returns the map of image Config for the registry version
-func (r *RegistryList) getImageConfigs() (map[string]Config, error) {
+func (r *RegistryList) getImageConfigs() (map[int]Config, error) {
 	switch r.K8sVersion.Segments()[0] {
 	case 1:
 		switch r.K8sVersion.Segments()[1] {
-		case 13:
-			return r.v1_13(), nil
-		case 14:
-			return r.v1_14(), nil
-		case 15:
-			return r.v1_15(), nil
-		case 16:
-			return r.v1_16(), nil
+		case 13, 14, 15, 16:
+			return nil, fmt.Errorf("version not supported for this build: %v", r.K8sVersion)
```

Comment:
🎉 I like this. Also, given that we are no longer supporting these earlier versions, can the files which define the `v1_13` to `v1_16` functions be removed?

Reply:
Yes they can!!!!!!!!! 🎉